lib/src/soloud.dart

```diff
@@ -148,10 +148,6 @@ class SoLoud {
       } else {
         debugIsolates('******** MAIN EVENT data: $data');
         if (data is StreamSoundEvent) {
-          debugPrint('@@@@@@@@@@@ SOUND EVENT: ${data.event}  '
-              'handle: ${data.handle}  '
-              'sound: ${data.sound}');
-
```

Comment:
These are helpful. I'd just do this using `log` instead of `debugPrint`. If you leave this in, I'll convert it when I do my pass of the codebase (hopefully today, tomorrow or on Monday).